velox/docs/functions/spark/comparison.rst

```diff
@@ -0,0 +1,56 @@
+=====================================
+Comparison Functions
+=====================================
+
+.. function:: between(x, min, max) -> boolean
+
+    Returns true if x is within the specified [min, max] range
+    inclusive. Supports TINYINT, SMALLINT, INTEGER, BIGINT, DOUBLE,
+    REAL types. The types of all arguments must be the same.
+
+.. function:: equalto(x, y) -> boolean
+
+    Returns true if x is equal to y. Supports all scalar types. The
+    types of x and y must be the same.
+
+.. function:: equalnullsafe(x, y) -> boolean
```

Comment:
I do not see equalnullsafe function in https://spark.apache.org/docs/2.4.1/api/sql/index.html#shiftright Is this a valid SparkSQL function?

Reply:
https://spark.apache.org/docs/latest/api/sql/index.html#_10 Here is the define of equalnullsafe.

Reply:
Got it. Thank you for the pointer.